nix/cardano-services/deployments/backend-deployment.yaml

```diff
@@ -0,0 +1,108 @@
+              value: "true"
+            - name: HANDLE_POLICY_IDS
+              value: f0ff48bbb7bbe9d59a40f1ce90e9e9d0ff5002ec48f232b49ca0fb9a
+            - name: HANDLE_PROVIDER_SERVER_URL
+              value: https://preview.api.handle.me
+            - name: LOGGER_MIN_SEVERITY
+              value: debug
+            - name: NETWORK
+              value: preview
+            - name: OGMIOS_SRV_SERVICE_NAME
+              value: dev-preview-cardano-stack.dev-preview.svc.cluster.local
+            - name: PAGINATION_PAGE_SIZE_LIMIT
+              value: "5500"
+            - name: POSTGRES_DB_DB_SYNC
+              value: cardano
+            - name: POSTGRES_HOST_DB_SYNC
+              value: dev-preview-dbsync-db
+            - name: POSTGRES_PASSWORD_DB_SYNC
+              valueFrom:
+                secretKeyRef:
+                  key: password
+                  name: cardano-owner-user.dev-preview-dbsync-db.credentials.postgresql.acid.zalan.do
+            - name: POSTGRES_POOL_MAX_DB_SYNC
+              value: "50"
+            - name: POSTGRES_PORT_DB_SYNC
+              value: "5432"
+            - name: POSTGRES_SSL_CA_FILE_DB_SYNC
+              value: /tls/ca.crt
+            - name: POSTGRES_SSL_DB_SYNC
+              value: "true"
+            - name: POSTGRES_USER_DB_SYNC
+              valueFrom:
+                secretKeyRef:
+                  key: username
+                  name: cardano-owner-user.dev-preview-dbsync-db.credentials.postgresql.acid.zalan.do
+            - name: SERVICE_NAMES
+              value: asset,chain-history,network-info,rewards,stake-pool,tx-submit,utxo
+            - name: TOKEN_METADATA_SERVER_URL
+              value: http://dev-preview-cardano-stack-metadata.dev-preview.svc.cluster.local
+            - name: USE_BLOCKFROST
+              value: "false"
+            - name: USE_KORA_LABS
+              value: "true"
+          image: 926093910549.dkr.ecr.us-east-1.amazonaws.com/cardano-services-server:aana7a352vdaz0rs41yhfh3s85inw004
+          livenessProbe:
+            httpGet:
+              path: /health
+              port: 3000
+            timeoutSeconds: 5
+          name: backend
+          ports:
+            - containerPort: 3000
+              name: http
+          resources:
+            limits:
+              cpu: 1500m
+              memory: 512Mi
+            requests:
+              cpu: 1000m
+              memory: 350Mi
+          securityContext:
+            runAsGroup: 0
+            runAsUser: 0
+          volumeMounts:
+            - mountPath: /tls
+              name: tls
+      imagePullSecrets:
+        - name: dockerconfigjson
+      volumes:
+        - csi:
+            driver: csi.cert-manager.io
+            readOnly: true
+            volumeAttributes:
+              csi.cert-manager.io/issuer-kind: ClusterIssuer
+              csi.cert-manager.io/issuer-name: root-ca
+              csi.cert-manager.io/key-usages: client auth
+          name: tls
```

Comment:
I am not a fan to have this set up for single environment only but I guess we can merge it and fix it later

I was curious how it would look like if this yaml was formatted properly. 108 lines turned into 50 lines. In case of cardano-stack we would also have to remove `resources`
```suggestion
apiVersion: apps/v1
kind: Deployment
metadata:
  labels:
    app: backend
spec:
  selector:
    matchLabels:
      app: backend
  template:
    metadata:
      labels:
        app: backend
    spec:
      containers:
        - args:
            - start-provider-server
          livenessProbe:
            httpGet:
              path: /health
              port: 3000
            timeoutSeconds: 5
          name: backend
          ports:
            - containerPort: 3000
              name: http
          resources:
            limits:
              cpu: 1500m
              memory: 512Mi
            requests:
              cpu: 1000m
              memory: 350Mi
          securityContext:
            runAsGroup: 0
            runAsUser: 0
          volumeMounts:
            - mountPath: /tls
              name: tls
      imagePullSecrets:
        - name: dockerconfigjson
      volumes:
        - csi:
            driver: csi.cert-manager.io
            readOnly: true
            volumeAttributes:
              csi.cert-manager.io/issuer-kind: ClusterIssuer
              csi.cert-manager.io/issuer-name: root-ca
              csi.cert-manager.io/key-usages: client auth
          name: tls
```

Reply:
Removed the lines which should not be here.  Instead, they should live together with overwrites since they are very mutable (just like OCI image)

Reply:
What's the reason for including them here @blaggacao ?

Reply:
This is as green-to-green as it gets (and just designed to proof and establish the pipeline).

I captured this too for timely follow up in [LW-8029](https://input-output.atlassian.net/browse/LW-8029).

Reply:
green-to-green does not mean that it should be in yaml, it just means that no additional changes should have been applied